examples/sai-i2s-passthru.rs

```diff
@@ -1,19 +1,16 @@
 // This demo code runs on the Electro Smith Daisy Seed board
 // https://www.electro-smith.com/daisy
+#![allow(unused_macros)]
 #![deny(warnings)]
-#![deny(unsafe_code)]
```

Comment:
Please leave this lint enabled, although I appreciate it has to be temporarily removed for debugging

Reply:
The log-itm code relies on an unsafe block. Unsafe cannot be denied if the utility logger is used the checks choke on it.

https://github.com/stm32-rs/stm32h7xx-hal/blob/a625d2b56a675dd2990056751301bb136338a86c/examples/utilities/logger.rs#L20

Reply:
Hmm yes good point. Let's leave it as-is for now then